apps/firebase/src/database-helper.ts

```diff
@@ -39,6 +39,7 @@ export interface RequestRecord {
   goldTxHash?: string
   tokens?: RequestedTokenSet
   authLevel: AuthLevel
+  timestamp: number
```

Comment:
im guessing this was leftover but i actually like it. if we can actually set it on each that would be great